feature-libs/quote/components/aware/quote-aware.component.html

```diff
@@ -0,0 +1 @@
+<ng-container> </ng-container>
```

Comment:
Is this file required?

Reply:
@ChristophHi, do we need this HTML file?

Reply:
Yes it is required but I can clean it up completely